docs/dashboards/chart-builder.md

```diff
@@ -176,3 +190,18 @@ Variables can be created in Report options "Manage variables".
 
 ![Manage Variables](../images/dashboards/variables.png)
 
+
+## Style
+
+In the style tab, you configure what the chart looks like. You can select bar, line, area and point 
+charts. In addition to chart type, you can choose a color from the palette and group color scheme. The main
```

Comment:
charts. In addition to chart type, you can choose a color from the palette and group color scheme. The main
=>
charts. In addition to the chart type, you can choose a color from the palette and group color scheme. The main